lib/ReactGridLayout.jsx

```diff
@@ -755,6 +761,10 @@ export default class ReactGridLayout extends React.Component<Props, State> {
 
   render(): React.Element<"div"> {
     const { className, style, isDroppable, innerRef } = this.props;
+    
+    if (innerRef) {
+      innerRef.current = this.ref.current;
+    }
```

Comment:
Hmm, maybe this should be moved to `componentDidMount` to ensure the `innerRef` hook is sync'd after render 🧐